tools/dep_updaters/update-base64.sh

```diff
@@ -2,6 +2,16 @@
 set -e
 # Shell script to update base64 in the source tree to a specific version
 
+update_base64_version(){
+  FILE_PATH="$BASE_DIR/src/base64_version.h"
```

Comment:
Do we need a function that's called once? I think it would be more readable to inline it.

Reply:
thought wrap in a function for readability.

Removed the function and added inline as suggested.